data/strings/strings.txt

```diff
@@ -23516,83 +23542,94 @@
+    ar:other = تم العثور على %d ملفًا. يمكنك رؤيتها بعد التحويل.
+    ar:two = تم العثور على %d ملفين. يمكنك رؤيتهما بعد التحويل.
+    az:one = %d fayl tapıldı. Onu dönüşümdən sonra görəcəksiniz.
+    az:other = %d fayl tapıldı. Onları dönüşümdən sonra görəcəksiniz.
     bg:one = Открит е %d файл. Можете да го видите след конвертирането.
     bg:other = Открити са %d файла. Можете да ги видите след конвертирането.
     ca:one = S'ha trobat %d fitxer. Podeu veure'l després de la conversió.
-    ca:other = S'han trobat %d fiters. Podeu veure'ls després de la conversió.
-    cs:few = %d byly nalezeny soubory. Uvidíte je po konverzi.
-    cs:one = %d soubor byl nalezen. Uvidíte ji po konverzi.
+    ca:other = S'han trobat %d fitxers. Podeu veure'ls després de la conversió.
+    cs:few = %d soubory byly nalezeny. Uvidíte je po konverzi.
+    cs:one = %d soubor byl nalezen. Uvidíte ho po konverzi.
     cs:other = Bylo nalezeno %d souborů. Uvidíte je po konverzi.
     da:one = %d fil blev fundet. Du vil se det efter konvertering.
-    da:other = Der er fundet %d filer. Du vil se dem efter konvertering.
-    de:one = %d Datei wurde gefunden. Sie werden sie nach der Konvertierung sehen.
-    de:other = %d Dateien wurden gefunden. Sie werden sie nach der Konvertierung sehen.
+    da:other = %d filer blev fundet. Du vil se dem efter konvertering.
+    de:one = %d Datei wurde gefunden. Sie können sie nach der Konvertierung sehen.
+    de:other = %d Dateien wurden gefunden. Sie können sie nach der Konvertierung sehen.
     el:one = Βρέθηκε %d αρχείο. Θα το δείτε μετά τη μετατροπή.
-    el:other = έχουν βρεθεί %d αρχεία. Θα τα δείτε μετά τη μετατροπή.
+    el:other = Βρέθηκαν %d αρχεία. Θα τα δείτε μετά τη μετατροπή.
     es-MX:one = Se encontró %d archivo. Lo verás después de la conversión.
-    es-MX:other = Se han encontrado %d archivos. Los verás después de la conversión.
-    es:one = Se encontró %d archivo. Lo verá después de la conversión.
-    es:other = Se han encontrado %d archivos. Los verá después de la conversión.
-    et:one = %d fail leiti. Näed seda peale teisendamist.
-    et:other = %d faili leiti. Näed neid peale teisendamist.
-    eu:one = fitxategi %d aurkitu da. Elkarrizketaren ondoren ikusiko duzu.
-    eu:other = %d fitxategi aurkitu dira. Elkarrizketaren ondoren ikusiko dituzu.
-    fa:one = %d فایل پیدا شد. بعد از تبدیل آن را می‌بینید.
-    fa:other = %d فایل پیدا شد. بعد از تبدیل آن‌ها را می‌بینید.
+    es-MX:other = Se encontraron %d archivos. Los verás después de la conversión.
+    es:one = Se encontró %d archivo. Lo verás después de la conversión.
+    es:other = Se encontraron %d archivos. Los verás después de la conversión.
+    et:one = %d fail leiti. Näed seda pärast teisendamist.
+    et:other = %d faili leiti. Näed neid pärast teisendamist.
+    eu:one = Aurkitu da %d fitxategi. Ikusiko duzu bihurketaren ondoren.
+    eu:other = %d fitxategi aurkitu dira. Ikusiko dituzu bihurketaren ondoren.
```

Comment:
Generally `es`, `pt`, `et`, `eu` should be in a good shape and proof-read already as we have active translators for these languages.
I hope AI won't make it worse.
It'd be good to mark these strings as "need review" in WL after the import.